changelog/24558.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:feature
-core/audit: add filter parameter when enabling an audit device, allowing filtering (using go-bexpr expressions) of audit entries written to the device's audit log
+**Audit Filtering**: Audit devices support go-bexpr-based filters.
```

Comment:
```suggestion
**Audit Filtering**: Audit devices support regular-expression-based filter rules to determine what is audit logged (powered by go-bexpr).
```

I'm more sure about the first part, and not 100% sure if that's the best way to describe the feature. What is logged? Which devices it gets logged to? This doesn't remove certain bits from the audit log output - that'll be redaction.

Reply:
RegEx is supported by go-bexpr, but not all expressions are in the RegEx syntax. How about this?

```suggestion
**Audit Filtering**: Audit devices support expression-based filter rules (powered by go-bexpr) to determine which entries are written to the audit log .```